rest/data/task.go

```diff
@@ -177,18 +95,9 @@ func (tc *DBTaskConnector) FindTasksByProjectAndCommit(project, commitHash, task
 	return res, nil
 }
 
-// SetTaskPriority changes the priority value of a task using a call to the
-// service layer function.
-func (tc *DBTaskConnector) SetTaskPriority(t *task.Task, user string, priority int64) error {
-	if t == nil {
-		return errors.New("task cannot be nil")
-	}
-	return model.SetTaskPriority(*t, priority, user)
-}
-
-// SetTaskPriority changes the priority value of a task using a call to the
+// SetTaskActivated changes the priority value of a task using a call to the
 // service layer function.
-func (tc *DBTaskConnector) SetTaskActivated(taskId, user string, activated bool) error {
+func SetTaskActivated(taskId, user string, activated bool) error {
```

Comment:
This  task is unnecessary; we can just call SetActiveStateById directly https://github.com/evergreen-ci/evergreen/blob/cfb0a7a16e8bf6b22d79f196b94eef3c37e1e1d5/model/task_lifecycle.go#L142